app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{#link-to "rentals.show" rentalUnit.slug}}
```

Comment:
OK.  We could link the title and within the listing component with a link-to, or we could create a toolbar above the listing with actions.  I think first options would be simplest and move us forward...